test/unit/migrations/migrator-test.js

```diff
@@ -35,6 +37,23 @@ const firstTimeState = {
   meta: { version: 0 },
   data: require('../../../app/scripts/first-time-state'),
 }
+describe('liveMigrations require list', () => {
```

Comment:
What does `liveMigrations` mean?

Reply:
`liveMigrations` are the migrations that are "live", e.g. listed in the index, primed to be run when the extension starts up.